### PR TITLE
Update runtime and libraries

### LINKED ISSUE #5: Update runtime to 19.08

### PR DESCRIPTION
**Pull request by lionirdeadman**
Fix #5

### CODE REVIEW

**Comment by lionirdeadman** [comment]
cc @hfiguiere

File: org.speed_dreams.SpeedDreams.yaml [hunk 1]
```diff
@@ -72,14 +74,14 @@ modules:
       - CXXFLAGS=-fPIC
     sources:
       - type: archive
-        url: http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
+        url: https://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
```
Comment (hfiguiere) [inline org.speed_dreams.SpeedDreams.yaml]: I get connection refused on this.

Could be another sourceforget hiccup.

File: org.speed_dreams.SpeedDreams.yaml [hunk 2]
```diff
@@ -72,14 +74,14 @@ modules:
...
-        url: http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
+        url: https://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
         sha256: 485b22bf6fdc0da067e34ead5e26f002b76326f6371e2ae006415dea6a380a32
 
   - name: enet
     sources:
       - type: archive
-        url: http://enet.bespin.org/download/enet-1.3.13.tar.gz
-        sha256: e36072021faa28731b08c15b1c3b5b91b911baf5f6abcc7fe4a6d425abada35c
+        url: http://enet.bespin.org/download/enet-1.3.17.tar.gz
```
Comment (hfiguiere) [inline org.speed_dreams.SpeedDreams.yaml:83]: no https?

File: org.speed_dreams.SpeedDreams.yaml (hunk 1 above)
Comment (son-link) [reply org.speed_dreams.SpeedDreams.yaml]: The website of the library does not have a secure connection, it is an old website, in fact it is a library that has been abandoned for years. I have looked at SourceForge and the only file there is from a release candidate of this version (1.8.5-rc1):
https://sourceforge.net/projects/plib/files/plib/

File: org.speed_dreams.SpeedDreams.yaml (hunk 2 above)
Comment (lionirdeadman) [reply org.speed_dreams.SpeedDreams.yaml:83]: It needs http otherwise there's a connection error.

File: org.speed_dreams.SpeedDreams.yaml (hunk 1 above)
Comment (lionirdeadman) [reply org.speed_dreams.SpeedDreams.yaml]: Yeah, I think I added https after building and didn't notice. I was going to say this but I'm compiling it again to make sure I didn't make another mistake.

**Comment by lionirdeadman** [comment]
cc @son-link Can it be merged? :sweat_smile:

**Comment by son-link** [comment]
Sorry, I've had a lot of work to do these days and completely forgot about it. 😅

**Pull request merged**